lib/jqlite.js

```diff
@@ -49,9 +49,16 @@
         if (typeof newElem == "string") {
           newElem = document.createElement(newElem);
         }
-        this.element.appendChild(newElem);
-        return new JQLite(newElem);
+        this.element[0].appendChild(newElem);
+        return new JQLite([newElem]);
       }
+
+      JQLite.prototype.remove = function() {
+        this.element.forEach(function(element) {
```

Comment:
only if it isn't supposed to be terminal. Honestly, returning the parent might be more useful, but returning the removed element might be more predictable.